diff -Nur old/src/conf_yacc.y new/src/conf_yacc.y
--- old/src/conf_yacc.y	2003-08-18 13:03:22.000000000 +0000
+++ new/src/conf_yacc.y	2003-12-13 19:31:05.000000000 +0000
@@ -156,6 +156,9 @@
        | TEND_DB {
 	  conferror("Error while reading configuration");
           }
+       | TEND_DBNOMD {
+	  conferror("Error while reading configuration");
+          }
        | error {
 	  conferror("Error while reading configuration");
 	  YYABORT;
