diff -urN aide-0.8.org/configure.in aide-0.8/configure.in
--- aide-0.8.org/configure.in	Wed Feb 27 09:24:28 2002
+++ aide-0.8/configure.in	Wed Feb 27 09:38:42 2002
@@ -189,17 +189,8 @@
 
 AC_CHECK_HEADERS(syslog.h)
 
-AIDE_USE_LOCALE="#"
-
-AC_ARG_WITH(locale,
-  [--with-locale   use locale stuff],
-  AC_CHECK_HEADERS(libintl.h,
-	AC_DEFINE(USE_LOCALE)
-	AIDE_USE_LOCALE=""
-        AC_DEFINE_UNQUOTED(LOCALEDIR,"$prefix/lib/locale"),)
-)
-
-AC_SUBST(AIDE_USE_LOCALE)
+ALL_LINGUAS="ru"
+AM_GNU_GETTEXT
 
 AC_ARG_WITH(syslog_ident,
   [--with-syslog_ident   Specify syslog ident],
@@ -379,6 +370,6 @@
 
 AC_DEFINE_UNQUOTED(AIDECOMPILEOPTIONS, "${compoptionstring}")
 
-AC_OUTPUT(Makefile src/Makefile po/Makefile doc/Makefile doc/aide.conf 
+AC_OUTPUT(Makefile src/Makefile intl/Makefile po/Makefile.in doc/Makefile doc/aide.conf 
 aide.spec)
 
