--- aide-0.15.1/src/conf_lex.l~	2010-08-08 19:39:31.000000000 +0200
+++ aide-0.15.1/src/conf_lex.l	2013-02-17 00:28:27.265336729 +0100
@@ -12,7 +12,7 @@
 
 %{
 
-#define YYDEBUG
+#define YYDEBUG 1
 
 /*
  * Copyright (C) 1999-2002,2004-2006,2010 Rami Lehti, Pablo Virolainen, Richard
--- aide-0.15.1/src/db_lex.l~	2010-08-08 19:39:31.000000000 +0200
+++ aide-0.15.1/src/db_lex.l	2013-02-17 00:32:07.953812738 +0100
@@ -45,7 +45,7 @@
 #define YY_DECL int db_scan(void)
 
 
-#define YYDEBUG
+#define YYDEBUG 1
 
 #include "aide.h"
 #include "conf_yacc.h"
