diff -Nur old/configure.in new/configure.in
--- old/configure.in	2003-12-13 19:43:39.000000000 +0000
+++ new/configure.in	2003-12-15 02:30:53.000000000 +0000
@@ -187,11 +187,11 @@
 AC_CHECK_FUNCS(readdir readdir_r stricmp ustat)
 AC_ARG_WITH(mmap,
 	[--without-mmap    Don't use mmap. Useful on some platforms with slow mmap],
-	if test "x$withval" = "xyes"; then 
+	[if test "x$withval" = "xyes"; then 
 		AC_CHECK_FUNCS(mmap)
-	fi
+	fi]
 	,
-	AC_CHECK_FUNCS(mmap)
+	[AC_CHECK_FUNCS(mmap)]
 )
 AC_CHECK_FUNC(vsnprintf,,
 	[AC_MSG_ERROR([Must have a system that supports vsnprintf.])]
@@ -241,12 +241,12 @@
 AC_MSG_CHECKING(for acl-support)
 AC_ARG_WITH(sun-acl,
   [--with-sun-acl	use ACL on solaris (no checking)],
-  AC_DEFINE(WITH_SUN_ACL)
+  [AC_DEFINE(WITH_SUN_ACL)
   AC_DEFINE(WITH_ACL)
   ACLLIB="-lsec"
-  AC_MSG_RESULT(yes)
+  AC_MSG_RESULT(yes)]
   ,
-  AC_MSG_RESULT(no)
+  [AC_MSG_RESULT(no)]
   )
 
 AC_SUBST(ACLLIB)
@@ -258,9 +258,10 @@
 AC_EGREP_CPP(sunisdefined,
     [#ifdef sun
 	sunisdefined
-    #endif],
-    AC_CHECK_FUNC(lstat64,
-	AC_CHECK_FUNC(stat64,
+    #endif
+    ],
+    [AC_CHECK_FUNC(lstat64,
+	[AC_CHECK_FUNC(stat64,
   		AC_DEFINE_UNQUOTED(AIDE_LSTAT_FUNC,lstat64)
   		AC_DEFINE_UNQUOTED(AIDE_STAT_FUNC,stat64)
   		AC_DEFINE_UNQUOTED(AIDE_FSTAT_FUNC,fstat64)
@@ -285,9 +286,9 @@
 		AC_DEFINE_UNQUOTED(AIDE_DIRENT_TYPE,dirent)
 		compoptionstring="${compoptionstring}WITH_LSTAT\\n"
 		compoptionstring="${compoptionstring}WITH_READDIR\\n"
-		)
+		)]
 	,
-	AC_DEFINE_UNQUOTED(AIDE_LSTAT_FUNC,lstat)
+	[AC_DEFINE_UNQUOTED(AIDE_LSTAT_FUNC,lstat)
 	AC_DEFINE_UNQUOTED(AIDE_STAT_FUNC,stat)
 	AC_DEFINE_UNQUOTED(AIDE_FSTAT_FUNC,fstat)
       	AC_DEFINE_UNQUOTED(AIDE_STAT_TYPE,stat)
@@ -298,10 +299,10 @@
   	AC_DEFINE_UNQUOTED(AIDE_READDIR_R_FUNC,readdir_r)
   	AC_DEFINE_UNQUOTED(AIDE_DIRENT_TYPE,dirent)
 	compoptionstring="${compoptionstring}WITH_LSTAT\\n"
-  	compoptionstring="${compoptionstring}WITH_READDIR\\n"
-	)	
+  	compoptionstring="${compoptionstring}WITH_READDIR\\n"]
+	)]
   ,
-  AC_DEFINE_UNQUOTED(AIDE_LSTAT_FUNC,lstat)
+  [AC_DEFINE_UNQUOTED(AIDE_LSTAT_FUNC,lstat)
   AC_DEFINE_UNQUOTED(AIDE_STAT_FUNC,stat)
   AC_DEFINE_UNQUOTED(AIDE_FSTAT_FUNC,fstat)
   AC_DEFINE_UNQUOTED(AIDE_STAT_TYPE,stat)
@@ -312,7 +313,7 @@
   AC_DEFINE_UNQUOTED(AIDE_READDIR_FUNC,readdir)
   AC_DEFINE_UNQUOTED(AIDE_READDIR_R_FUNC,readdir_r)
   AC_DEFINE_UNQUOTED(AIDE_DIRENT_TYPE,dirent)
-  compoptionstring="${compoptionstring}WITH_READDIR\\n"
+  compoptionstring="${compoptionstring}WITH_READDIR\\n"]
   )
 
 
@@ -365,32 +366,29 @@
 fi
 
 AC_ARG_WITH([gnu-regexp],
-  [--with-gnu-regexp  use the bundled GNU regexp(use only if necessary)],
-  [AC_DEFINE(REGEX, 1)]
-,
-  [AC_CHECK_FUNCS(regexec regcomp,,
-	[AC_DEFINE(REGEX,1)])]
+  [--with-gnu-regexp  use the bundled GNU regexp (use only if necessary)],
+  [AC_DEFINE(REGEX, 1)],
+  [AC_CHECK_FUNCS(regexec regcomp,, [AC_DEFINE(REGEX,1)])]
 )
 
-AC_ARG_WITH(gcrypt,
-  [--with-gcrypt  use gcrypt library],
-  AC_DEFINE(WITH_GCRYPT)
+AC_ARG_WITH(gcrypt,  [--with-gcrypt  use gcrypt library],
+  [AC_DEFINE(WITH_GCRYPT)
   AC_CHECK_HEADER(gcrypt.h,,
-  	AC_MSG_ERROR(You don't have libgcrypt properly installed.
-Install it if you need one.)
+  	[AC_MSG_ERROR(You don't have libgcrypt properly installed.
+Install it if you need one.)]
 )
   CRYPTLIB="-lgcrypt"
-  compoptionstring="${compoptionstring}WITH_GCRYPT\\n"
+  compoptionstring="${compoptionstring}WITH_GCRYPT\\n"]
   ,
   [with_gcrypt=no])
 
-AC_CHECK_LIB(mhash,mhash_get_block_size,
+#AC_CHECK_LIB(mhash,mhash_get_block_size,
   CRYPTLIB="-lmhash"
   compoptionstring="${compoptionstring}WITH_MHASH\\n"
   AC_DEFINE(WITH_MHASH)
-,
-	AC_MSG_ERROR(You must have libmhash properly installed.)
-)
+#,
+#	AC_MSG_ERROR(You must have libmhash properly installed.)
+#)
 
 AC_SUBST(CRYPTLIB)
 
@@ -400,12 +398,11 @@
   if test "x$withval" = "xmd5" ;then
       AC_DEFINE(CONFIGHMACTYPE,MHASH_MD5)
   else 
-  if test "x$withval" = "xsha1" ;then
-      AC_DEFINE(CONFIGHMACTYPE,MHASH_SHA1)
-  else 
-      echo "Valid parameters for --with-confighmactype are md5, sha1"
-      exit 1
-  fi
+    if test "x$withval" = "xsha1" ;then
+        AC_DEFINE(CONFIGHMACTYPE,MHASH_SHA1)
+    else 
+        AC_MSG_ERROR(Valid parameters for --with-confighmactype are md5, sha1)
+    fi
   fi
 ,
 )
@@ -447,12 +444,11 @@
   if test "x$withval" = "xmd5" ;then
       AC_DEFINE(DBHMACTYPE,MHASH_MD5)
   else 
-  if test "x$withval" = "xsha1" ;then
-      AC_DEFINE(DBHMACTYPE,MHASH_SHA1)
-  else 
-      echo "Valid parameters for --with-dbhmactype are md5, sha1"
-      exit 1
-  fi
+    if test "x$withval" = "xsha1" ;then
+        AC_DEFINE(DBHMACTYPE,MHASH_SHA1)
+    else 
+        AC_MSG_ERROR(Valid parameters for --with-dbhmactype are md5, sha1)
+    fi
   fi
 ,
 )
@@ -532,6 +528,10 @@
 
 AC_DEFINE_UNQUOTED(AIDECOMPILEOPTIONS, "${compoptionstring}")
 
-AC_OUTPUT(Makefile src/Makefile intl/Makefile po/Makefile.in doc/Makefile doc/aide.conf 
-aide.spec)
+AC_OUTPUT([
+	    Makefile 
+	    src/Makefile 
+	    doc/Makefile
+	    doc/aide.conf 
+	    aide.spec])
 
