diff -Naur aide-0.7/configure.in aide-0.7-p/configure.in
--- aide-0.7/configure.in	Fri Nov 23 16:38:09 2001
+++ aide-0.7-p/configure.in	Fri Nov 23 16:38:30 2001
@@ -289,7 +289,7 @@
 fi
 
 AC_ARG_WITH(gnu-regexp,
-  [--with-gnu-regexp  use the bundled GNU regexp(use only if necessary)],
+  [--with-gnu-regexp  use the bundled GNU regexp (use only if necessary)],
   AC_DEFINE(REGEX, 1)
 ,
   AC_CHECK_FUNCS(regexec regcomp,,
diff -Naur aide-0.7/configure.in aide-0.7-p/configure.in
--- aide-0.7/configure.in	Fri Nov 23 16:46:36 2001
+++ aide-0.7-p/configure.in	Fri Nov 23 16:46:27 2001
@@ -289,11 +289,11 @@
 fi
 
 AC_ARG_WITH(gnu-regexp,
-  [--with-gnu-regexp  use the bundled GNU regexp (use only if necessary)],
-  AC_DEFINE(REGEX, 1)
-,
-  AC_CHECK_FUNCS(regexec regcomp,,
-	AC_DEFINE(REGEX,1))
+  "--with-gnu-regexp  use the bundled GNU regexp (use only if necessary)",
+  [AC_DEFINE(REGEX, 1)],
+  [AC_CHECK_FUNCS(regexec regcomp,,
+	[AC_DEFINE(REGEX,1)]
+  )]
 )
 
 AC_CHECK_HEADER(gcrypt.h,,
diff -Naur aide-0.9/Makefile.am aide-0.9-p/Makefile.am
--- aide-0.9/Makefile.am	Tue Jun  4 10:29:57 2002
+++ aide-0.9-p/Makefile.am	Sat Nov 23 10:49:00 2002
@@ -17,7 +17,7 @@
 # Boston, MA 02111-1307, USA.
 #
 
-SUBDIRS = src doc po
+SUBDIRS = src doc po intl
 
 CLEANFILES = *~
 
diff -Naur aide-0.9/src/Makefile.am aide-0.9-/src/Makefile.am
--- aide-0.9/src/Makefile.am	Wed Mar  6 14:12:12 2002
+++ aide-0.9-/src/Makefile.am	Sat Nov 23 10:55:14 2002
@@ -46,7 +46,7 @@
 		util.c		\
 		aide.c
 
-INCLUDES = -I$(prefix)/include -I$(top_srcdir)/include
+INCLUDES = -I$(top_srcdir)/include -I$(includedir)
 
 LDADD = @CRYPTLIB@ @ACLLIB@
 
