diff -Nur old/Makefile.am new/Makefile.am
--- old/Makefile.am	2003-01-16 10:37:34.000000000 +0000
+++ new/Makefile.am	2003-12-13 19:21:12.000000000 +0000
@@ -17,7 +17,7 @@
 # Boston, MA 02111-1307, USA.
 #
 
-SUBDIRS = src doc po
+SUBDIRS = src doc po intl
 
 CLEANFILES = *~
 
diff -Nur old/src/Makefile.am new/src/Makefile.am
--- old/src/Makefile.am	2003-10-29 10:10:57.000000000 +0000
+++ new/src/Makefile.am	2003-12-13 19:22:44.000000000 +0000
@@ -46,7 +46,7 @@
 		util.c		\
 		aide.c
 
-INCLUDES = -I$(prefix)/include -I$(top_srcdir)/include
+INCLUDES = -I$(top_srcdir)/include -I$(includedir)
 
 LDADD = @CRYPTLIB@ @ACLLIB@
 
