--- aide-0.13.1/configure.in~	2007-11-05 01:47:02.000000000 +0100
+++ aide-0.13.1/configure.in	2007-11-05 01:49:09.610684250 +0100
@@ -282,23 +282,8 @@
 
 AC_CHECK_HEADERS(syslog.h inttypes.h fcntl.h)
 
-AC_ARG_WITH([locale],
-	[AC_HELP_STRING([--with-locale],
-		[use locale stuff])],
-	[],
-	[with_locale=no]
-)
-
-AS_IF([test "x$with_locale" != xno],
-	[AC_CHECK_HEADERS(libintl.h,
-		[AC_DEFINE(USE_LOCALE,1,[Define if LOCALE support should be used])
-		AIDE_USE_LOCALE=""
-		compoptionstring="${compoptionstring}WITH_LOCALE\\n"
-dnl        	AC_DEFINE_UNQUOTED(LOCALEDIR,"$prefix/lib/locale",[Localedir to use])],
-	)]
-)
-
-AC_SUBST(AIDE_USE_LOCALE)
+ALL_LINGUAS=""
+AM_GNU_GETTEXT
 
 AC_ARG_WITH([syslog_ident],
 	AC_HELP_STRING([--with-syslog-ident],
