diff -Naur aide-0.7/configure.in aide-0.7-p/configure.in
--- aide-0.7/configure.in	Fri Nov 23 16:38:09 2001
+++ aide-0.7-p/configure.in	Fri Nov 23 16:38:30 2001
@@ -289,7 +289,7 @@
 fi
 
 AC_ARG_WITH(gnu-regexp,
-  [--with-gnu-regexp  use the bundled GNU regexp(use only if necessary)],
+  [--with-gnu-regexp  use the bundled GNU regexp (use only if necessary)],
   AC_DEFINE(REGEX, 1)
 ,
   AC_CHECK_FUNCS(regexec regcomp,,
diff -Naur aide-0.7/configure.in aide-0.7-p/configure.in
--- aide-0.7/configure.in	Fri Nov 23 16:46:36 2001
+++ aide-0.7-p/configure.in	Fri Nov 23 16:46:27 2001
@@ -289,11 +289,11 @@
 fi
 
 AC_ARG_WITH(gnu-regexp,
-  [--with-gnu-regexp  use the bundled GNU regexp (use only if necessary)],
-  AC_DEFINE(REGEX, 1)
-,
-  AC_CHECK_FUNCS(regexec regcomp,,
-	AC_DEFINE(REGEX,1))
+  "--with-gnu-regexp  use the bundled GNU regexp (use only if necessary)",
+  [AC_DEFINE(REGEX, 1)],
+  [AC_CHECK_FUNCS(regexec regcomp,,
+	[AC_DEFINE(REGEX,1)]
+  )]
 )
 
 AC_CHECK_HEADER(gcrypt.h,,
